tests/config.coffee

```diff
@@ -0,0 +1,3 @@
+module.exports =
+	publishKey: 'pub-c-02e5848f-c9c5-4e6a-8586-268c44635766'
+	subscribeKey: 'sub-c-4a67af8a-049f-11e7-b09b-0619f8945a4f'
```

Comment:
The keys used in production are anyway published to the entire world: https://api.resin.io/config

The channel name itself is here the _secret secret_.